llama-index-core/llama_index/core/vector_stores/types.py

```diff
@@ -74,6 +75,34 @@ class FilterOperator(str, Enum):
     ALL = "all"  # Contains all (array of strings)
     TEXT_MATCH = "text_match"  # full text match (allows you to search for a specific substring, token or phrase within the text field)
     CONTAINS = "contains"  # metadata array contains value (string or number)
+    LIKE = "like"  # metadata field contains value in substring (string)
+
+
+class FilterOperatorFunction(str, Enum):
```

Comment:
Is this specific to milvus?

Reply:
It is. PgVector has something similar (because it's just PostreSQL CRUD operations), but it looks like managed vector dbs like Pinecone don't support this. It's also not necessary for my use case. I was just adding it in to expand on Milvus's capabilities

Reply:
Since this doesn't really generalize to other vector dbs, I would say to just remove it (and if we want a way to add this to milvus vector store specifically, then we can figure that out in that package)

Reply:
Reverted this